pages/404.tsx

```diff
@@ -0,0 +1,17 @@
+import { useRouter } from "next/router";
+import { LandingNav, LandingFooter } from "@/Components/components";
+
+const NotFoundPage = () => {
+  return (
+    <>
+      <LandingNav error404Check={false} />
```

Comment:
```suggestion
      <LandingNav error404Check={true} />
```

this prop should be true from this page.

Reply:
done